pages/styles/about.css

```diff
@@ -0,0 +1,88 @@
+
+* {
+    font-family: sans-serif;
+}
+
+header {
+    justify-content: left;
+    display: flex;
+    height: 300px;
+    order: 2;
+    flex-direction: column;
+    flex-shrink: 0;
+    width: 200px; 
+    align-self: flex-start;
+    margin: 40px;
+}
+
+
+h3 {
+    justify-content: center;
+    display: flex;
+    order: 3;
+    flex-direction: column;
+    flex-shrink: 0;
+    width: 600px; 
+    flex-grow: 2;
+    flex-wrap: wrap;
+    height: 80px;
+    align-content: space-between;
+    margin: 50px 50px 50px 250px;  
+}
+
+nav {
+    bottom: 250px;
+    display: flex;
+    flex-direction: row-reverse;
+    font-size: 25px;
+    position: relative;
+    left: 740px;
+    width: 500px;
+    margin: auto; 
+    order: 1;
+}
+
+
+li {
+    display: inline-block;
+    list-style: none;
+    margin: 10px 10px; 
+}
+
+nav a {
+    text-decoration: none;
+    color: #c6fbf9;
+    font-weight: bold;
+}  
+
+nav a:hover{
+    color: #fbd7c6;
+}
+
+h1 {
+    font-size: 100px;
+    color: #c6d0fb;
+    margin-bottom: 10px;
+    color: #ffebe0;
```

Comment:
This `color` property is repeated on line 64 as well. In the end this means this later one on line 66 will win out.